arrays/python/count-inversions.py

```diff
@@ -23,9 +25,22 @@ def count_split_inv(arr, left, right):
     elif ridx == rsize:
       arr[i] = left[lidx]
       lidx += 1
-  return split_inv;
+  return split_inv
+
 
-def count_inversions(arr):
+def count_inversions(arr) -> int:
+  """
+  Sort arr and return the number of inversions required.
+
+  >>> numbers
+  [8, 2, 1, 5, 7, 3, 9, 2, 0, 1]
+  >>> count_inversions(numbers)
+  30
```

Comment:
This should be `28` the problem in line 15 also affects this

Reply:
Perfect!